test/Directory.Build.props

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <DeveloperBuildTestTfms>netcoreapp2.2</DeveloperBuildTestTfms>
     <StandardTestTfms>$(DeveloperBuildTestTfms)</StandardTestTfms>
-
+    <NoWarn>618</NoWarn>
```

Comment:
I'm assuming it's way too much to use the `#pragma`s in the test projects on individual lines? Is it reasonable to do it per-file? Or at least per-project?

Reply:
This would only stay in 2.2 branch, I'm going to remove both obsolete methods and this directive in master.

Reply:
I'll try switching to per-file.